README.md

```diff
@@ -8,19 +8,18 @@ Nonetheless, username/password is now **deprecated** and you **must** rely on AP
 
 # Prometheus exporter for Confluent Cloud Metrics API
 
-A simple prometheus exporter that can be used to extract metrics from [Confluent Cloud Metric API](https://docs.confluent.io/current/cloud/metrics-api.html).
-By default, the exporter will be exposing the metrics on [port 2112](http://localhost:2112)
-To use the exporter, the following environment variables need to be specified:
-
-* `CCLOUD_API_KEY`: The API Key created with `ccloud api-key create --resource cloud`
-* `CCLOUD_API_SECRET`: The API Key Secret created with `ccloud api-key create --resource cloud`
-
-`CCLOUD_API_KEY` and `CCLOUD_API_SECRET` environment variables will be used to invoke the https://api.telemetry.confluent.cloud endpoint.
+* A simple prometheus exporter that can be used to extract metrics from [Confluent Cloud Metric API](https://docs.confluent.io/current/cloud/metrics-api.html).
+* By default, the exporter will be exposing the metrics on [port 2112](http://localhost:2112)
+* To use the exporter, the following environment variables need to be specified: `CCLOUD_API_KEY` and `CCLOUD_API_SECRET`.
+    * The API Key and Secret created with `ccloud api-key create --resource cloud`.
+    * These environment variables will be used to invoke the `https://api.telemetry.confluent.cloud` endpoint.
+    * Place those variables into `./ccloud_exporter.env`
```

Comment:
Hum, I am not sure we should do this change. It encourages people to rely on a `.env` file to store secrets. 
It is fine if you want to use this file to push it as a Kubernetes secret, but for non-Kubernetes environment, having plaintext API keys on a non-encrypted file could be an issue.

Reply:
I don't see this as an issue, it is an improvement;`ccloud_exporter.env` is in `.gitignore` so never pushed to repo neither shared. These kind of files are normally in Vault, Key Vault, AWS Secrets Manager, ...

Reply:
You might be right, I guess I will sleep & read on this point ;)
I will merge the PR, then do the minor modification in another commit.